theano/gof/tests/test_op.py

```diff
@@ -233,6 +235,28 @@ def c_code(self, node, name, inputs, outputs, sub):
             self.assertRaises((NotImplementedError, utils.MethodNotDefined),
                               thunk)
 
+    def test_no_make_node(self):
+        class IncOne(Op):
+            """An Op without make_node"""
+            __props__ = ()
+            itypes = [T.fmatrix]
+            otypes = [T.fmatrix]
+
+            def perform(self, node, inputs, outputs):
+                input, = inputs
+                output, = outputs
+                output[0] = input + 1
+
+        x_input = T.fmatrix('x')
+        o = IncOne()(x_input)
+
+        # Confirming that make_node method is implemented
```

Comment:
The comment do not check if make_node is implemented. I don't understand what this try except is testing. The only missing tests I think of is to compile a Theano function and execute it and make sure it return the right value.